internal/droplet/droplet_tools.go

```diff
@@ -526,8 +526,44 @@ func (d *DropletTool) enablePrivateNetworkingByTag(ctx context.Context, req mcp.
 }
 
 // Tools returns a list of tool functions
```

Comment:
I think this is not accurate anymore

```suggestion
// Tools returns a list of tool functions
```